developer-tools.md

```diff
@@ -274,17 +262,6 @@ A binary incompatibility reported by MiMa might look like the following:
 [error] filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.SomeClass.this")
 ```
 
-If you open a pull request containing binary incompatibilities anyway, Jenkins
```

Comment:
What does Github actions do her?

Reply:
I would just remove these references, all of the Jenkins references. I think it's OK, rather than rewriting them in terms of GHA. We can add back docs later